src/fixture.ts

```diff
@@ -29,7 +29,7 @@ export interface Boo { b_oo: null }
  * https://github.com/standard/eslint-config-standard-with-typescript/issues/110
  */
 // Inline callbacks don't need return types:
-setTimeout(() => {}, 1)
+setTimeout(() => { console.log() }, 1)
```

Comment:
Suddenly this started failing. And it should. `@typescript-eslint/no-empty-function`.